tests/config_aggregator/main.tf

```diff
@@ -0,0 +1,31 @@
+provider aws {
+  region = "us-east-1"
+}
+
+module config_aggregator {
+  source = "../../"
+
+  aggregator = {
+    name = "tardigrade-config-aggregator-${random_string.this.result}"
+    tags = {}
+    account_aggregation_source = {
+      account_ids = [data.aws_caller_identity.current.account_id]
+      all_regions = true
+      regions     = null
+    }
+    organization_aggregation_source = null
```

Comment:
This assumes you don't want/need to test organization_aggregation_source?

Reply:
Right. Testing org-level features is tricky. We don't currently have a dedicated AWS Organizations account for tardigrade testing. We have member accounts for testing, but not a dedicated org account. Probably not a bad idea to get one. Will need to think about how we'd go about that...